tutorials/settings.gradle.kts

```diff
@@ -0,0 +1,2 @@
+rootProject.name = "tutorials"
```

Comment:
1. Maybe we put script not under 'tutorials' folder, but under scripts/check-tutorials? So `tutorials` remains clean.
2. I think it is better to add gradlew or gradlew.bat in the folder.
3. Also we need to add `build` and `.gradle` folder into .gitignore

Reply:
You forgot to move `gradle/wrapper` folder :)